src/index.js

```diff
@@ -72,7 +72,7 @@ function OfflineTable(props) {
   }
 
   function filterData(e) {
-    const filterText = e.target.value
+    const filterText = e.target.value;
```

Comment:
I think its better to lowercase filterText once here instead of having to do it everywhere.